frb_codegen/src/generator/dart/ty_enum.rs

```diff
@@ -6,6 +6,16 @@ use crate::utils::BlockIndex;
 
 type_dart_generator_struct!(TypeEnumRefGenerator, IrTypeEnumRef);
 
+impl<'a> TypeEnumRefGenerator<'a> {
+    fn dart_implements(&self) -> &'static str {
+        if self.ir.is_exception {
+            "implements FrbException"
+        } else {
+            ""
+        }
+    }
+}
```

Comment:
uber-nit: seems this is duplicated with `TypeStructRefGenerator::dart_implements`, so maybe we can extract a function like `fn dart_maybe_implements_exception(is_exception: bool)` etc